packages/native-stack/src/views/NativeStackView.native.tsx

```diff
@@ -190,13 +216,20 @@ const SceneView = ({
             false
           : gestureEnabled
       }
+      homeIndicatorHidden={homeIndicatorHidden}
```

Comment:
```suggestion
      homeIndicatorHidden={autoHideHomeIndicator}
```
After discussing with @satya164 we've decided to name this `autoHideHomeIndicator` as it's impossible to hide it permanently. It just prefers to stay hidden.

Reply:
Ok, keep in mind that we need to update this in my documentation PR as well.